bindings/Sofa/src/SofaPython3/Sofa/Simulation/Submodule_Simulation.cpp

```diff
@@ -69,6 +69,12 @@ PYBIND11_MODULE(Simulation, simulation)
         return node ? py::cast(node.get()) : py::none();
     }, sofapython3::doc::simulation::load);
     simulation.def("unload", [](Node* n){ sofa::simulation::getSimulation()->unload(n); }, sofapython3::doc::simulation::unload);
+    simulation.def("step", [](Node *n, int n_steps, SReal dt=0.0){
+        for (int i = 0; i < n_steps; i++)
+        {
+            sofa::simulation::getSimulation()->animate(n, dt); //Execute one timestep. If dt is 0, the dt parameter in the graph will be used
+        }
+    }, sofapython3::doc::simulation::step, py::arg("root_node"), py::arg("n_steps") = 1, py::arg("dt") = 0.0);
```

Comment:
I use this function to "decouple" the "frequency" of the simulation.
For example I am just interested in an observation every second, but want my dt to be much smaller.
I would like a nice and fast cpp loop to do that for me instead of doing that through many python calls.

I will change the name right away. :)